README.md

```diff
@@ -739,13 +739,35 @@ const requestData: TimeFilterFastProtoRequest = {
     lat: 51.508824,
     lng: -0.167093,
   }],
-  transportation: 'driving+ferry',
+  transportation: {
+    mode: 'driving+pt',
+    details: { // only available for 'driving+pt` and 'pt'. Optional
+      walkingTimeToStation: 1800, // optional, available for 'pt' and 'driving+pt'
+      drivingTimeToStation: 1800, // optional, available for 'driving+pt'
+      parkingTime: 900 // optional, available for 'driving+pt'
+    }
+  },
+  // Simpler, if details are not needed
+  // transportation: 'driving+ferry', 
```

Comment:
Is this really the best way to implement this? Could it be defined with types?
@MockusTravelTime What do you think?